website/blog/2019-08-21-introduction.md

```diff
@@ -0,0 +1,88 @@
+
+Covering complex use cases will often require writing a lot of business logic that needs to be later manually removed by developers who want to reuse sample application for their needs.
+
+## Sample apps working with your business logic
+
+If you ever worked with simple sample application you may recall how hard is to adjust those to real business cases. 
+
+As an alternative to sample apps, developers can rely on frameworks that provide a high level of abstraction. Technologies that offer rapid application development often come at the cost of the maintenance and flexibility that can seriously limit the extensibility of your application server. 
+
+Over the years our team focused on developing scalable GraphQL based servers and clients and we are often were asking ourselves:
+
+> Can we build an application template that can offer production-ready capabilities and yet is simple enough to work with without extensive learning?
+
+> Can we provide our data model as input to the GraphQL engine and benefit autogenerated data access methods?
+
+Answers for these questions led us to the creation of the new GraphQL based ecosystem called Graphback.
+
+## Introducing Graphback
+
+![Alt Text](https://thepracticaldev.s3.amazonaws.com/i/b58zov9fctzk44qzwrln.png)
+
+The target of Graphback is to make sample applications working with your business logic by utilizing application templates and code generation. Developers can provide GraphQL schema as input and the command-line client will generate data access methods on top of the chosen stack (called template) and databases we support. 
+
+Graphback generates Node.JS application with an entire codebase that is editable and can be later deployed to your own server or cloud provider.
+By default, Graphback will follow the best patterns and include proven open-source libraries provided by the vibrant GraphQL community. When working with relational databases Graphback will allow users to manage database schema. The real power of Graphback is its code generation engine that can be extended for different needs. 
+
+![diagram](https://thepracticaldev.s3.amazonaws.com/i/c9q6rvbb9v534073cq6j.png)
+
+
+## Graphback to resolve major issues in GraphQL community
+
+Graphback helps developers to boost their getting started the experience with GraphQL. Graphback CLI offers different workflows supporting fetching data from the database and rest endpoints. 
+Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of the libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the GraphQL entire ecosystem. 
```

Comment:
```suggestion
Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the entire GraphQL ecosystem.
```

Reply:
Should "flavors" be "flavours"? It depends on if we want US English or Irish/British.